src/styles/Header/AdminNav.js

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import {size} from "../variables";
+import MEDIA from "../mediaTemplates";
+
+const Container = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  background-color: #171c1d;
+  color: white;
+  font-size: 8px;
+  font-weight: bold;
+
+  ul {
+    list-style: none;
+    display: inline-flex;
+    padding-left: 10px;
+  }
+
+  li {
+    font-size: 13px;
+    margin-right: ${size.tiny};
+    padding: 2px 7px;
+    background: #0c0e0e;
```

Comment:
Use a variable here. If needed you can add to this list, but it will keep the colors consistent throughout the app. 

https://github.com/open-sauced/open-sauced/blob/master/src/styles/variables.js#L1-L19